package.json

```diff
@@ -12,15 +12,32 @@
     "url": "https://github.com/eventualbuddha/keysim.js"
   },
   "scripts": {
-    "build": "gobble build dist -f",
+    "prebuild": "mkdir -p dist",
+    "build:node": "npm run prebuild && babel lib/keysim.es > dist/keysim.node.js",
+    "build:browser": "npm run prebuild && browserify . --standalone keysim > dist/keysim.browser.js",
+    "build": "npm run build:node && npm run build:browser",
     "test": "npm run test:node && npm run test:browser",
-    "test:browser": "./node_modules/karma/bin/karma start karma.conf.js",
-    "test:node": "mocha -R spec --recursive test"
+    "test:browser": "karma start --single-run",
+    "test:node": "mocha --require babel-register -R spec --recursive test",
+    "prepublish": "in-publish && npm test && npm run build || not-in-publish"
+  },
+  "browser": "lib/keysim.es",
+  "browserify": {
+    "transform": [
+      "babelify"
+    ],
+    "ignore": [
+      "jsdom"
+    ]
+  },
+  "babel": {
+    "presets": [
+      "es2015"
+    ],
+    "plugins": [
+      "transform-class-properties"
+    ]
   },
-  "files": [
```

Comment:
I was confused by this. You can read my confusion here:
https://github.com/npm/npm/issues/10924

I ended up removing this and adding an empty `.npmignore` file.